pyproject.toml

```
@@ -23,7 +23,7 @@ dependencies = [
     "ipywidgets>=8.0.6",
     "voila>=0.4,<0.5",
     "pyyaml>=5.4.1",
-    "specutils>=1.9",
+    "specutils>=2.0",
```

Same as https://github.com/glue-viz/glue-astronomy/pull/100#pullrequestreview-2056205321

```suggestion
    "specutils>=2.0.dev",
```

---

But then we'll see all the test failures that I haven't updated to fix 😭 😆